core/app/models/spree/product/scopes.rb

```diff
@@ -63,9 +63,10 @@ def self.property_conditions(property)
     #
     #   SELECT COUNT(*) ...
     add_search_scope :in_taxon do |taxon|
-      includes(:classifications).
-      where("spree_products_taxons.taxon_id" => taxon.self_and_descendants.pluck(:id)).
-      order(Spree::Classification.arel_table[:position].asc)
+      includes(:classifications)
+        .where('spree_products_taxons.taxon_id' => taxon.self_and_descendants.pluck(:id))
+        .select('spree_products_taxons.position')
```

Comment:
I think this `select` statement refers to the same table and field of the following `order` statement, they're just written in two different way. 

Should it make more sense having them written in the same way? So it's evident to everyone we're referencing the same field.

Reply:
Sure, let me have a look!